vertical-pod-autoscaler/docs/faq.md

```diff
@@ -244,3 +245,15 @@ The `--webhook-labels` parameter for the VPA admission-controller can be used to
 When running in EKS with Cilium, the EKS API server cannot route traffic to the overlay network. The VPA admission-controller
 Pods either need to use host networking or be exposed through a service or ingress.
 See the [Cilium Helm installation page](https://docs.cilium.io/en/stable/installation/k8s-install-helm/) for more info.
+
+### Why does VPA fail to create a Pod when resource limits are explicitly set to 0?
+VPA currently handles requests=0 and limits=0 inconsistently with Kubernetes, leading to invalid Pod configurations. Here’s the breakdown:
+
+| Scenario                | Kubernetes Behavior                          | VPA Behavior                                  |
+|-------------------------|----------------------------------------------|----------------------------------------------|
+| `requests=0`+`limits=0`| Disables resource fields (BestEffort QoS)    | Disables limits but recommends non-zero requests (**QoS conflict**) |
```

Comment:
In this scenario the VPA sets limits to 0 and requests to the recommendation.
This is an invalid scenario, I think it's worth describing here.